CHANGELOG.md

```diff
@@ -17,6 +17,15 @@
   convert that file to plain JS (`//= require active_admin/base` if you
   didn't add any stuff to it).
 
+### Enhancements
+
+* Add your own content to the site `<head>`, like analytics [#5590][] by [@buren][]
```

Comment:
You're following the same change log style as the one present in your local branch, but it does not match what's currently in master. I'm suggesting the change to match the style in master, but ideally you would rebase this PR to make sure everything works fine against the latest code in master.

```suggestion
* Add your own content to the site `<head>`, like analytics [#5590] by [@buren]
```